include/CoreGraphics/CGContext.h

```diff
@@ -193,11 +193,11 @@ COREGRAPHICS_EXPORT void CGContextStrokeEllipseInRect(CGContextRef c, CGRect rec
 COREGRAPHICS_EXPORT void CGContextStrokeLineSegments(CGContextRef c, const CGPoint* points, size_t count);
 COREGRAPHICS_EXPORT bool CGContextIsPathEmpty(CGContextRef c);
 
-COREGRAPHICS_EXPORT CGPoint CGContextGetPathCurrentPoint(CGContextRef c) STUB_METHOD;
```

Comment:
@msft-Jeyaram good question.  The answer is - very likely yes.  At least CGContextGetPathCurrentPoint for example - but it is a very thin wrapper around a different API - CGPathGetCurrentPoint.